### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by guilegatan** [LibreBooking/app#438]
A colleague recently reported they had an accessibility audit of their webservices and LibreBooking did not fully comply.
I still have to get the full report, but I just wanted to ask if this could be tackled in the foreseeable future? Of course depending on how severe the report grades the missing features.

Thanks in advance.

**Comment by labmecanicatec** [comment]
We believe that with the project of updating to Bootstrap 5, accessibility has improved. It would be good to review the full report to take it into account.

**Comment by guilegatan** [comment]
Sorry for the delay. I had to read the audit summary and translate the relevant points to English.
These things where criticised:

- some buttons and icons have no description (aka "alt"-info)
- on most subpages H1 headings are missing
- tables have no defined headers (aka <TH> sections)
- color coding is used for information (i.e. on the schedule-page for Open, Blocked, Reserved, My Reservation, etc.)
- nearly no explicit keyboard control possible (i.e. omitting use of a mouse)
- no skiplinks
- distance between certain clickable items is too narrow
- form-elements have no label or it is omitted by screenreaders, and forms have no visible description
- some labels only have placeholder text
- multiple use of same ID elements
- success messages are not focused or read automatically by screenreaders

This sounds a lot to me, and I am not sure if these things can be realized at all. Especially because the summary omitts specific links or screenshots, so I cannot be 100% sure of certain problems.

Thanks in advance to whoever is willing to help with these issues!
